mods/d2k/rules/defaults.yaml

```diff
@@ -446,18 +446,22 @@
 	Inherits: ^Building
 	Inherits@selection: ^SelectableCombatBuilding
 	WithSpriteTurret:
+		RequiresCondition: !build-incomplete
 	AttackTurreted:
 	RenderRangeCircle:
 	DetectCloaked:
 		Range: 1c768
 	-GivesBuildableArea:
-	-WithMakeAnimation:
-	-WithCrumbleOverlay:
+	WithMakeAnimation:
+		BodyNames: make
 	-Capturable:
-	-WithSpriteBody:
+	WithSpriteBody:
+		Name: make
+		Sequence: null
 	Sellable:
 		RequiresCondition: !being-demolished
 	WithWallSpriteBody:
+		RequiresCondition: !build-incomplete
```

Comment:
This unfortunately introduces a visual glitch when the turret is placed on top of a wall - the adjacent segments change state as if the wall was sold, then change back once the make animation has completed. We either need another hacky invisible `WithWallSpriteBody` or a dummy `IWallConnector` trait to keep active during the make animation.

Reply:
On second thought, scratch that. My suggestion here looks even glitchier than the PR has now.